src/modules/movies-count.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const movieCount = document.querySelector('.movie-count');
+
+const count = async () => {
+  try {
+    const response = await axios.get('https://api.tvmaze.com/shows/1/episodes');
+    const result = response.data;
+    const movies = result.splice(0, 12);
+    movieCount.innerHTML = movies.length.toString();
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
+export default count;
```

Comment:
- Good job done so far, but you have not correctly implemented the movie test counter, according to the project requirements, `A counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page`. Please implement your counter to count the number of movies displayed, similar to what you did for the `comment` counter. Please fix this 🙏

Reply:
- [ ] @Team Great work implementing the changes requested, however as previously requested kindly consider changing how you have implemented the movies count method, it's supposed to use DOM elements as you did for the comments counter function in commentCount.js file.